### ISSUE TITLE
How to modify SVR loss function?

### ISSUE THREAD

**Issue by iN8mare** [cjlin1/libsvm#206]
I am working on a research project and I want to change the usual loss function of SVR by simply adding a hyperparameter that can penalize certain predicted values more compared to others.

For example, if the predicted value is greater than some k, then i want to increase the loss function by some hyperparameter alpha.

How can i implement or change this? 
I tried looking up in scikit learn but it's not possible to do this. 
Any help would be appreciated.

**Comment by cjlin1** [comment]
It may not be easy to do this in libsvm. You must derive the dual problem first and see if the current solver can solve the new optimization problem. You might want to write the loss as a continuous and almost differentiable function, and then apply a tool with autodiff (e.g., pytorch)
